sonic-syseepromd/scripts/syseepromd

```diff
@@ -115,6 +122,59 @@ class DaemonSyseeprom(daemon_base.DaemonBase):
 
         return True
 
+    def post_values_to_state_db(self, key, values):
+        try: 
+            state_db = daemon_base.db_connect(STATE_DB)
+            for k,v in values.items():
+                state_db.hset(key, k, str(v))
+        except Exception as e:
+            self.log_error(f'writing {k}:{v} to db failed with {e}')
```

Comment:
if exception occurs during line 127, the log would raise another exception because k,v does not exist.